pyroomacoustics/doa/tools_fri_doa_plane.py

```diff
@@ -1385,9 +1383,11 @@ def dirac_recon_ri_half_parallel(G, a_ri, K, M, max_ini=100):
     # generate all the random initialisations
     c_ri_half_all = np.random.randn(sz_coef, max_ini)
 
-    res_all = jl.Parallel(n_jobs=thread_num)(
-        jl.delayed(partial_dirac_recon)(c_ri_half_all[:, loop][:, np.newaxis])
-        for loop in range(max_ini))
```

Comment:
would `joblib` reduce computational time significantly? if so, it might be worth making a variable `joblib_available` like for `matplotlib`

Reply:
It was there only for historical reason. It was limited to one thread. This doesn't change performance at all. We tried it in the past and it didn't improve all that much.